src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -75,6 +75,48 @@
 
         return defaultBsPopperConfig;
     };
+    const getTextWidth = (text, fontProps = {}) => {
+        const styleProps = {
+            position: 'absolute',
+            left: '-1000px',
+            'white-space': 'nowrap',
+            ...fontProps,
+        };
+        const tag = doc.createElement('div');
+
+        tag.innerHTML = text;
+
+        for (const key in styleProps) {
+            tag.style[key] = styleProps[key];
+        }
+
+        doc.body.appendChild(tag);
+
+        const { width: tagWidth } = tag.getBoundingClientRect();
+
+        doc.body.removeChild(tag);
+
+        return tagWidth;
+    };
+    const isTitleEllipsized = (node) => {
+        const title = node.title || node.dataset.originalTitle;
+        const { scrollWidth, offsetWidth } = node;
+
+        if (scrollWidth === offsetWidth) {
+            const { width: nodeWidth } = node.getBoundingClientRect();
+            const computedNodeStyles = getComputedStyle(node);
+            const fontProps = {
```

Comment:
you should probably add other font styles as well https://www.w3schools.com/cssref/pr_font_font.php
You're missing `font-variant` and `font-style`